File: 0001 Code Chef/0015 Election Hopes/implementation/0008 python3/election_hopes.py
#!/bin/python3


if __name__ == "__main__":
    string_input = input()
    string_array = string_input.split(" ")
    x = int(string_array[0])
    y = int(string_array[1])

    if(x >= 2*y):
        print("YES")
    else:
        print("NO")

File: 0001 Code Chef/0015 Election Hopes/implementation/0004 java/ElectionHopes.java
import java.util.Scanner;
public class ElectionHopes{
    public static void main(String[] args){
        Scanner  console = new Scanner(System.in);
        String stringInput = console.nextLine();
        String[] stringArray = stringInput.split(" ");
        int x = Integer.parseInt(stringArray[0]);
        int y = Integer.parseInt(stringArray[1]);
        if(x >= 2*y){
            System.out.println("YES");
        }else{
            System.out.println("NO");
        }
    }
}